app/views/users/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Listing users</h1>
+
+<%= form_tag users_path, method: :get do %>
+<div>
+  <%= text_field_tag :campo, params[:name_search], placeholder: "Search by name", autofocus: true, style: "width: 200px; height: 25px;" %>
+  <%= date_field_tag :inicial_date, Date.today.beginning_of_month, autofocus: true, style: "width: 120px; margin-left: 3px;" %>
+  <%= date_field_tag :final_date, Date.today.end_of_month, autofocus: true, style: "width: 120px; margin-left: 3px;" %>
+  <%= submit_tag t('.index', :default => "Search"), style: "height: 35px; margin-left: 3px" %>
+</div>
```

Comment:
This `<div>` should be indented.. :P
You can create some css classes and avoid the use of `style`